data/json/items/comestibles/carnivore.json

```
@@ -1212,7 +1220,7 @@
     "proportional": { "price_postapoc": 0.5 },
     "fun": -20,
     "extend": { "flags": [ "BAD_TASTE" ] },
-    "vitamins": [ [ "mutant_toxin", 50 ], [ "meat_allergen", 1 ] ]
+    "vitamins": [ [ "vitC", 3 ], [ "calcium", 0 ], [ "iron", 10 ], [ "mutant_toxin", 50 ], [ "meat_allergen", 1 ] ]
```

```suggestion
    "vitamins": [ [ "vitC", 3 ], [ "iron", 10 ], [ "mutant_toxin", 50 ], [ "meat_allergen", 1 ] ]
```

This should be unnecessary? The default is 0 anyway, and currently it's not possible to overwrite the vitamins carried by ``copy-from`` anyway (and I would presume that when we do get the ability to overwrite them, it would be by overwriting the entire field)

I am assuming this is directly copied from the normal raw brain, and to be fair it shouldn't have that calcium defined either.

---

It's a copy, and I think there are more copies zero entries.